src/components/constants/url.js

```diff
@@ -1,4 +1,17 @@
 const BASE_URL = `${process.env.NEXT_PUBLIC_BASE_URL}`;
 export const SUBSCRIBE_TO_CHALLENGE_URL = `${BASE_URL}/challenges/subscribe`;
+export const TASKS_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/tasks/self`;
+export const LOGIN_URL = 'https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97';
+export const USER_PROFILE_URL = 'https://my.realdevsquad.com/profile';
+export const HOME_URL = 'https://realdevsquad.com';
+export const WELCOME_URL = 'https://welcome.realdevsquad.com';
+export const EVENTS_URL = 'https://www.realdevsquad.com/events.html';
+export const CRYPTO_URL = 'https://crypto.realdevsquad.com';
+export const STATUS_URL = 'https://status.realdevsquad.com';
+export const MEMBERS_URL = 'https://members.realdevsquad.com';
 export const CHALLENGES_URL = `${BASE_URL}/challenges`;
-export const USER_SELF = `${BASE_URL}/users/self`;
\ No newline at end of file
+export const USER_SELF = `${BASE_URL}/users/self`;
+export const DEFAULT_AVATAR = '/Avatar.png';
+export const RDSLogo = '/RDSLogo.png';
```

Comment:
Why isn't this using snake casing like other constants

Reply:
Good observation. Will update it